guides/v2.2/ui_comp_guide/components/ui-datecolumn.md

```diff
@@ -7,23 +7,48 @@ The DateColumn component implements a table column that displays dates.
 
 ## Configuration options
 
-<table>
-  <tr>
-    <th>Option</th>
-    <th>Description</th>
-    <th>Type</th>
-    <th>Default</th>
-  </tr>
-  <tr>
-    <td><code>dateFormat</code></td>
-    <td>Date format for the displayed column's values.</td>
-    <td>String</td>
-    <td><code>MMM d, YYYY h:mm:ss A</code></td>
-  </tr>
-</table>
+| Option | Description | Type | Default |
+| --- | --- | --- | --- |
+| `dateFormat` | Date format for the displayed column's values. | String | `MMM d, YYYY h:mm:ss A` |
```

Comment:
Is the format of the default value correct?

Reply:
I noticed that too, but the original PR was just converting the table from HTML to markdown here, so I let it go. But since you called it out, I checked, and the answer is yes.

```
    return Column.extend({
        defaults: {
            dateFormat: 'MMM d, YYYY h:mm:ss A',
            calendarConfig: []
        },
```